bpf/lib/nodeport.h

```diff
@@ -13,6 +13,7 @@
 #include "lb.h"
 #include "common.h"
 #include "overloadable.h"
+#include "egress_policies.h"
```

Comment:
nit: are there any policies that are not egress gw policies, that could be characterized as egress policies?

Reply:
SRv6 egress policies are coming :smiley: